package/utils/audit/Makefile

```diff
@@ -105,5 +154,31 @@ define Package/libaudit/install
 	$(CP) $(PKG_INSTALL_DIR)/etc/libaudit.conf $(1)/etc/
 endef
 
+define Package/libauparse/install
+	$(INSTALL_DIR) $(1)/usr/lib
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libauparse.so.* $(1)/usr/lib/
```

Comment:
Did you test this? Above at libaudit/install should be more specific or you get the following error:

```
Collected errors:
 * check_data_file_clashes: Package libauparse wants to install file /home/user/src/openwrt/openwrt/build_dir/target-aarch64_cortex-a53_musl/root-mediatek/usr/lib/libauparse.so.0
	But that file is already provided by package  * libaudit
 * check_data_file_clashes: Package libauparse wants to install file /home/user/src/openwrt/openwrt/build_dir/target-aarch64_cortex-a53_musl/root-mediatek/usr/lib/libauparse.so.0.0.0
	But that file is already provided by package  * libaudit
 * opkg_install_cmd: Cannot install package libauparse.
 * check_data_file_clashes: Package libauparse wants to install file /home/user/src/openwrt/openwrt/build_dir/target-aarch64_cortex-a53_musl/root-mediatek/usr/lib/libauparse.so.0
	But that file is already provided by package  * libaudit
 * check_data_file_clashes: Package libauparse wants to install file /home/user/src/openwrt/openwrt/build_dir/target-aarch64_cortex-a53_musl/root-mediatek/usr/lib/libauparse.so.0.0.0
	But that file is already provided by package  * libaudit
```

Reply:
Good point. I have only imported this and have not changed the make source.
However, I have not installed it via opkg.
I will fix it

Reply:
fixed https://github.com/openwrt/openwrt/pull/4136/files#diff-1f32d39f9d0396e56263fac69e37d243910bbc377250f2a9bbcfe74ac684863fR152